Sources/Sentry/SentryOptions.m

```diff
@@ -708,4 +709,136 @@ - (void)setEnablePreWarmedAppStartTracing:(BOOL)enablePreWarmedAppStartTracing
 
 #endif // SENTRY_UIKIT_AVAILABLE
 
+#if defined(DEGUG)
+- (NSString *)description
+{
+    NSMutableArray<NSString *> *optionsDescription = [NSMutableArray<NSString *> array];
```

Comment:
`m`: I think we must find a way to ensure we add future properties here; otherwise, it can be misleading. One idea is to count the number of properties of SentryOptions with something like
```objc
void printPropertiesOfClass(Class cls) {
    unsigned int count;
    objc_property_t *properties = class_copyPropertyList(cls, &count);

    for (unsigned int i = 0; i < count; i++) {
        const char *propertyName = property_getName(properties[i]);
    }

    free(properties);
}
```

and then compare it to the size of `optionsDescription`. If they don't match, a test fails.

Reply:
I wrote a totally dynamic solution.